README.md

```diff
@@ -60,6 +61,3 @@ The single run test accuracy of this PyTorch version is 85.84%, while the author
 The authors implementations uses 8 GPU and can thus accomodate a batch of 64 samples while the present implementation is single GPU and is in consequence limited to 20 instances on a K80 for memory reasons. In our test, increasing the batch size from 8 to 20 samples increased the test accuracy by 2.5 points. A better accuracy may be obtained by using a multi-GPU setting (on the TO-DO list).
```

Comment:
Can you also remove this TODO mention related to multi-GPU and maybe add a mention of your single-run test?